dev-ml/camlp5/camlp5-7.10.ebuild

```diff
@@ -0,0 +1,71 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib findlib eutils
+
+MY_P=${P%_p*}
+echo $MY_P
+DESCRIPTION="A preprocessor-pretty-printer of ocaml"
+HOMEPAGE="https://camlp5.github.io/"
+SRC_URI="https://github.com/camlp5/camlp5/archive/rel$(ver_rs 1- '').tar.gz -> ${P}.tar.gz"
+echo $SRC_URI
+S="${WORKDIR}/${PN}-rel$(ver_rs 1- '')"
+echo $S
+LICENSE="BSD"
+SLOT="0/${PV}"
+KEYWORDS="~alpha ~amd64 ~ppc ~x86"
+IUSE="doc +ocamlopt"
+
+DEPEND="
+	>=dev-lang/ocaml-3.10:=[ocamlopt?]
```

Comment:
There are no ocaml-3* ebuilds in the tree, so this is always satisfied and can be lifted off. Place the USE flag condition to ocaml dependency below.